app/models/dns.rb

```diff
@@ -0,0 +1,11 @@
+class Dns < Parameter
```

Comment:
This class name is quite generic, especially I'm afraid it can collide in the future with Foreman core (there's Orchestration::DNS there already). Could you namespace it under ForemanDocker, so it'd be `ForemanDocker::Dns`?